### ISSUE TITLE
Conan Package Tools 0.28.0 breaks compatibility with Conan 1.14

### ISSUE THREAD

**Issue by santhonisz** [conan-io/conan-package-tools#385]
### Description of Problem, Request, or Question
Release notes state that Conan Package Tools 0.28.0 is compatible with Conan <= 1.16. however upon updating from 0.27.0 we are getting the below failure (seems to be related to changes made for #379) 

### Environment Details
* Conan Package Tools Version: **0.28.0**
* Operating System: **Windows**
* Conan version: **conan 1.14.5**
* Python version: **python 3.6**

### Build logs (Include if Available)
```[09:56:41]	Version: 0.28.0
[09:56:41]	 >> Branch detected
[09:56:41]	   >> master
[09:56:41]	
[09:56:41]	 >> Info
[09:56:41]	   >> Redefined channel by CI branch matching with 'master$', setting CONAN_CHANNEL to 'stable'
[09:56:41]	Traceback (most recent call last):
[09:56:41]	  File "build.py", line 39, in <module>
[09:56:41]	    exclude_vcvars_precommand = True
[09:56:41]	  File "C:\buildAgent\.venv\shared\lib\site-packages\cpt\packager.py", line 192, in __init__
[09:56:41]	    conanfile = load_cf_class(os.path.join(self.cwd, self.conanfile), self.conan_api)
[09:56:41]	  File "C:\buildAgent\.venv\shared\lib\site-packages\cpt\packager.py", line 33, in load_cf_class
[09:56:41]	    remotes = conan_api._cache.registry.load_remotes()
[09:56:41]	AttributeError: 'RemoteRegistry' object has no attribute 'load_remotes'```

**Comment by danimtb** [comment]
Hi @santhonisz,

This seems an issue with the compatibility with older Conan versions. The method ``load_remotes`` was not available in the registry class in Conan 1.14.5 yet.

We will try to fix it and release a patch in the following days.

@uilianries could take care of this, please?

**Comment by uilianries** [comment]
@danimtb  Yes, I gonna take a look
